--- build_qemu_support.sh	2017-11-05 10:25:32.000000000 +0800
+++ build_qemu_support.sh	2018-05-31 21:32:51.742000000 +0800
@@ -56,7 +56,7 @@
 fi
 
 
-for i in libtool wget python automake autoconf sha384sum bison iconv; do
+for i in libtoolize wget python automake autoconf sha384sum bison iconv; do
 
   T=`which "$i" 2>/dev/null`
 
@@ -131,6 +131,8 @@
 patch -p1 <../patches/elfload.diff || exit 1
 patch -p1 <../patches/cpu-exec.diff || exit 1
 patch -p1 <../patches/syscall.diff || exit 1
+patch -p1 <../patches/configure.diff || exit 1
+patch -p1 <../patches/memfd.c.diff || exit 1
 
 echo "[+] Patching done."
 
